wpilibjExamples/src/main/java/org/wpilib/examples/hatchbotinlined/subsystems/DriveSubsystem.java

```diff
@@ -92,10 +88,10 @@ public void setMaxOutput(double maxOutput) {
   }
 
   @Override
-  public void initSendable(SendableBuilder builder) {
-    super.initSendable(builder);
+  public void updateTelemetry(TelemetryTable table) {
```

Comment:
```suggestion
  public void logTo(TelemetryTable table) {
```
Ditto throughout example subsystems. (Some have been updated, but not all have)